lms/product/product.py

```diff
@@ -34,9 +34,6 @@ class Routes:
     oauth2_refresh: Optional[str] = None
     """Refreshing OAuth 2 tokens."""
 
-    list_group_sets: Optional[str] = None
```

Comment:
Again, this is not part of the product, all product will use the same view/route.